client/packages/invoices/src/Returns/CustomerDetailView/Footer/Footer.tsx

```diff
@@ -38,12 +38,13 @@ const createStatusLog = (invoice: CustomerReturnFragment) => {
   if (statusIdx >= 0) {
     statusLog[InvoiceNodeStatus.New] = invoice.createdDatetime;
   }
-  if (statusIdx >= 2) {
+  if (statusIdx >= 1) {
     statusLog[InvoiceNodeStatus.Picked] = invoice.pickedDatetime;
   }
-  if (statusIdx >= 3) {
+  if (statusIdx >= 2) {
     statusLog[InvoiceNodeStatus.Shipped] = invoice.shippedDatetime;
   }
+  // Skipping delivered
```

Comment:
Why are we skipping received? 🤔 Should it not follow the same flow as Inbound since items are shipped to use, so we want to confirm that it has been delivered then received?

Also statusIdx has been changed to `  const statusIdx = allStatuses.findIndex(s => invoice.status === s);`
but this file is still using inboundStatuses

Reply:
Can we note this down to have a discussion somewhere. I think it's good to be consistent?

Reply:
I'm okay with keeping this as is for now~ but would like to discuss in the near future